DigitalLearningSolutions.Data/Services/NotificationService.cs

```diff
@@ -53,8 +53,8 @@ public async Task SendUnlockRequest(int progressId)
 
             var baseUrlConfigOption = refactoredTrackingSystemEnabled
                 ? configuration.GetAppRootPath()
-                : configService.GetConfigValue(ConfigHelper.CurrentSystemBaseUrlName);
-            if (baseUrlConfigOption == null)
+                : configuration.GetCurrentSystemBaseUrl();
+            if (string.IsNullOrEmpty(baseUrlConfigOption))
             {
                 throw new ConfigValueMissingException(
                     configService.GetConfigValueMissingExceptionMessage(

```

Comment:
Yeah, it could be helpful to throw an exception and say which config value is missing in the message.

Reply:
cool should it be a named Exception or just Exception? I've made another configValue MissingException in the Data project but that seems a bit confusing when we have another exception with the same name?

Reply:
You're alright to use the existing ConfigValueMissingException class, but rather than using a configService method to create it, just pass in your own message.

Reply:
Feel free to move the exception class out of that file and into the Exceptions folder while you're at it (Rider has a shortcut which takes care of the guff for you - think it's F6 by default).